src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -125,11 +125,18 @@ during compilation:
 [target.x86_64-fuchsia]
 
 rustflags = [
-    "-Lnative", "<SDK_PATH>/arch/x64/sysroot/lib",
-    "-Lnative", "<SDK_PATH>/arch/x64/lib"
+    "-Lnative=<SDK_PATH>/arch/x64/sysroot/lib",
+    "-Lnative=<SDK_PATH>/arch/x64/lib"
 ]
 ```
 
+These options configure the following:
+
+* `-Lnative=<SDK_PATH>/arch/x64/sysroot/lib`: Link against Fuchsia libraries from
+  the SDK
+* `-Lnative ${SDK_PATH}/arch/${ARCH}/sysroot/lib`: Link against Fuchsia kernel
```

Comment:
To clarify, the sysroot is a term for the language (C) runtime for a given platform. The C runtime happens to use syscalls, so that's where the libzircon stub is defined.

Reply:
```suggestion
* `-Lnative=${SDK_PATH}/arch/${ARCH}/sysroot/lib`: Link against Fuchsia kernel
```